src/SDL.c

```diff
@@ -485,9 +485,20 @@ luaopen_SDL(lua_State *L)
 	tableSetInt(L, -1, "VERSION_MAJOR", ver.major);
 	tableSetInt(L, -1, "VERSION_MINOR", ver.minor);
 	tableSetInt(L, -1, "VERSION_PATCH", ver.patch);
+	lua_newtable(L);
 
-	tableSetInt(L, -1, "VERSION_BINDING", VERSION_BINDING);
```

Comment:
Removing these fields would be a breaking change, even if they probably don't see use in practice;

I suggest hardcoding them to "4" and "1" respectively, so old code can recognize it as a "newer but compatible version", while new code can use the "binding" field to get the real version in the new numbering system.